tests/components/emulated_hue/test_hue_api.py

```diff
@@ -301,7 +301,7 @@ def test_put_light_state_fan(hass_hue, hue_client):
         blocking=True)
 
     # Emulated hue converts 0-100% to 0-255.
-    level = 23
+    level = 43
```

Comment:
This test was previously passing incorrectly. since 23 is within 0-33, it should have been low, but was asserted as medium since it was passing through the light domain branches. Updated the test now that the logic is fixed.